### PR TITLE
DEV-3395-new go exercise return a

### PR DESCRIPTION
**Pull request by zainabdnaya**

### CODE REVIEW

File: subjects/returna/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+## returna
+
+### Instructions
+
+Write a program that takes a `string` as an argument, if the string contains the letter `"a"` then print `"Contains the letter"` followed by a newline.
```
Comment (tiagocollot) [inline subjects/returna/README.md:5]: Instruction fix: "...if the `string` contains the letter `a` then print `Contains the letter` followed by a newline `\n`."

File: subjects/returna/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+- If there's no character `"a"` in the string, the program writes `"!(Contains the letter)"` followed by a newline.
```
Comment (tiagocollot) [inline subjects/returna/README.md:6]: Should be "...character `a` in the `string`, the program writes `!(Contains the letter)` followed by a newline `\n`."

File: subjects/returna/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+## returna
+
+### Instructions
+
+Write a program that takes a `string` as an argument, if the string contains the letter `"a"` then print `"Contains the letter"` followed by a newline.
+- If there's no character `"a"` in the string, the program writes `"!(Contains the letter)"` followed by a newline. 
+- If the number of parameters is not 1, the program displays a newline.
```
Comment (tiagocollot) [inline subjects/returna/README.md:7]: Missing `\n` after 'newline'.

**Review by tiagocollot — CHANGES_REQUESTED** [review]

File: subjects/returna/README.md (hunk 3 above)
Comment (jotapero) [reply subjects/returna/README.md]: fixed

File: subjects/returna/README.md (hunk 2 above)
Comment (jotapero) [reply subjects/returna/README.md]: fixed

File: subjects/returna/README.md (hunk 1 above)
Comment (jotapero) [reply subjects/returna/README.md]: fixed

**Review by tiagocollot — APPROVED** [review]
LGTM ⚡️

**Review by jotapero — APPROVED** [review]
LGTM :alien:

**Comment by jotapero** [comment]
Closed pull request and opened new one with correct files, conflicts were too many

**Pull request closed without merging**